src/20_Components/amp-social-share.html

```diff
@@ -90,5 +95,41 @@
 
   <amp-social-share type="pinterest" data-param-media="https://ampbyexample.com/img/amp.jpg"></amp-social-share>
 
+  <!-- #### Native Share Dialog (Chrome on Android) -->
+  <!--
+  If a couple of conditions are satisfied, the `system` type will display a native share
+  UI. (That is, exactly the same UI that's available to native applications, reflecting
+  the user's sharing preferences, the applications they have installed, and so on.)
+  
+  The conditions are:
+  
+    * The user must be viewing the AMP using Chrome on Android. (The only supported browser as of February 2017.)
+    * An appropriate token (origin-specific) must be embedded on the site to activate the feature. ([Apply for a token](https://goo.gl/GR5YLI); [more information about the process](https://developers.google.com/web/updates/2016/10/navigator-share).)
+    
+  For this example, we've embedded the tokens via `<meta>` elements [in the document `<head>`](#setup).
+
+  Here's what it looks like if conditions are right:
+
+  <amp-img alt="Animation showing native sharing sheet appearing" src="/img/system-share.gif" width="270" height="480"></amp-img>
+  -->
+
+  <amp-social-share type="system"></amp-social-share>
+
+  <!--
+  This type supports one option: if `data-mode="replace"` is specified, then all
+  other share icons will be suppressed. (That is, the only share icon the user
+  will see is the system share icon.)
+
+  Since it's not possible to demo this feature within the context of this
```

Comment:
Oh, what I was trying to convey is that it's not possible to include the string `<amp-social-share type="system" data-mode="replace"></amp-img>` anywhere in the source of the sample, because when opened on an Android device, this will cause all other social share buttons to disappear, including those in the "Basic Usage" section, etc.